shared/src/persistence/dynamo/trialSessions/addCaseToHearing.js

```diff
@@ -0,0 +1,34 @@
+const { put } = require('../../dynamodbClientService');
+const { updateTrialSession } = require('./updateTrialSession');
+
+/**
+ * addCaseToHearing
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.docketNumber docket number of the case to add
+ * @param {object} providers.trialSession trial session to add as a hearing
+ * @returns {Promise} the promise of the call to persistence
+ */
+exports.addCaseToHearing = async ({
+  applicationContext,
+  docketNumber,
+  trialSession,
+}) => {
+  return await Promise.all([
+    // Create mapping record
+    put({
+      Item: {
+        pk: `case|${docketNumber}`,
+        sk: `hearing|${trialSession.trialSessionId}`,
+        ...trialSession,
+      },
+      applicationContext,
+    }),
+    // update trial session
+    updateTrialSession({
+      applicationContext,
+      trialSessionToUpdate: trialSession,
+    }),
+  ]);
```

Comment:
Yeah, it should not matter at this point. Promise.all() does not guarantee a sequential order of ops. We use this same pattern in case creation, and other areas of the app.